src/diagnostics/build.ts

```diff
@@ -56,7 +56,9 @@ export class CompileOutputConsumer implements OutputConsumer {
       case 'remark':
         return vscode.DiagnosticSeverity.Information;
       }
-      throw new Error('Unknown diagnostic severity level: ' + p);
+      // tslint:disable-next-line
+      console.warn('Unknown diagnostic severity level: ' + p);
```

Comment:
Why not use log.warning instead? I think we can probably use a logger for this file, or accept a Logger in the constructor like CMakeBuildConsumer.

Reply:
I think it would be too noisy on localized systems.  We have issues with the problem matchers not supporting translated versions of "error", "warning", etc coming from the compiler.